src/memote/checks/Reactions.jl

```diff
@@ -0,0 +1,76 @@
+"""
+    module Reactions
+
+A module testing reaction properties.
+"""
+module Reactions
+
+using COBREXA
+using DocStringExtensions
+
+import ..Config
+import ..Biomass: findall_biomass_reactions
+import ..GPRAssociation: reaction_has_sensible_gpr
+
+"""
+$(TYPEDSIGNATURES)
+
+Return a list of all reactions that are duplicated.
+"""
+function duplicate_reactions(model::MetabolicModel)
```

Comment:
naming rant: you're using mostly imperative functions (`find_xxxs(model)` actually means `xxxs_taken_from(model)`). Here I had hard time to guess whether this is making duplicates of the reactions (conforms with the rest of the naming) or is a predicate for 2 reactions (not enough parameters) or actually just lacks the usual `find_all_`. I'm actually in favor of the "functional" naming scheme as used here, without the imperatives. Perhaps better specify where you searching for the reactions (`duplicated_reactions_in_model`) or so.

No need to fix tho, this is a nitpick. :D

Reply:
fixing